config/routes.rb

```diff
@@ -1,13 +1,11 @@
 Rails.application.routes.draw do
   devise_for :users, controllers: { confirmations: 'confirmations' }
 
-  get 'welcome/index'
+  resources :registered_applications
```

Comment:
This route looks good... I do not see the controller argument here though. Is that a different branch?

Reply:
I'm not sure what you mean by controller argument?